src/util/misc.ts

```diff
@@ -141,3 +141,25 @@ export function getScopeUnit(
 
     return node.vScope.vScope;
 }
+
+/**
+ * Given an array `arr` return a new array with all the elements of
+ * `arr` but without duplicates. Elements are compared with the `===`.
+ *
+ * @param arr
```

Comment:
We can remove the `@param` to keep it short.

Reply:
Fixed in 6d2b8b6.